data/assets/css/components/_event-summary.scss

```diff
@@ -14,9 +14,12 @@
 .event-summary {
   position: relative;
   justify-self: left;
-  margin-bottom: $space-large;
+  width: 100%;
+  margin-bottom: $space-large/2;
```

Comment:
instead of dividing, try it with @space-medium - https://github.com/VLCTechHub/VLCTechHub-site/blob/a6e70150326410852befa87199816ca6dc8a8e7a/data/assets/css/abstract/_variables.scss#L12